### DISCUSSION TITLE
Build instructions and player

### DISCUSSION THREAD

**Discussion by pandom79** [LMS-Community/slimserver#885] [General]
Are there build instructions for linux amd64?
Actually, I built it but i'm not sure if it has been properly done.
I am not able to play music because there is not a player.
How can i resolve it ?
Thanks in advance.

**Comment by philippe44** [comment]
Build what?

**Comment by pandom79** [comment]
slimserver ?

I created my build script from https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=logitechmediaserver

**Comment by michaelherger** [comment]
There are pre-built packages available: https://downloads.slimdevices.com/LogitechMediaServer_v8.3.1/

**Comment by pandom79** [comment]
> 

In my case, it should be https://downloads.slimdevices.com/LogitechMediaServer_v8.3.1/logitechmediaserver-8.3.1.tgz.
Anyway, i think properly build the package but i don't understand why it says "No player found".
Should I install anything else ?

**Comment by michaelherger** [comment]
LMS is a music server application, not a player. You'd need a hardware player like a Squeezebox or a software emulator (https://github.com/ralph-irving/squeezelite). But if you didn't know that, then the LMS ecosystem probably isn't for you.

**Comment by pandom79** [comment]
Resolved.
Thanks.

**Comment by TSEACie** [comment]
Posting here in the hope of getting in touch with @ralph-irving re SqueezePlay for Windows.  Specifically, is it possible to Autoplay X Favourite on Startup following a network or power outage or PC reboot?  Thank you!
